<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBookForm</class>
 <widget class="QDialog" name="EditBookForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="auteurLabel">
      <property name="text">
       <string>Auteur</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="auteurLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="genreLabel">
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="genreLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dateDAjoutLabel">
      <property name="text">
       <string>Date d'ajout</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dateDAjoutLineEdit"/>
    </item>
    <item row="4" column="0">
     <layout class="QFormLayout" name="formLayout_3"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="titreLabel">
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="titreLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
